reflex/components/component.py

```diff
@@ -932,6 +989,30 @@ def _has_event_triggers(self) -> bool:
                     return True
         return False
 
+    @classmethod
+    def _iter_parent_classes_with_method(cls, method: str) -> Iterator[Type[Component]]:
```

Comment:
Does this affect compile time noticeably at all?

Reply:
on reflex-web, there is no change in compile time 😅, i expect this iter operation to be quite fast; although it does have to look at all parent classes in the mro chain, it doesn't actually call the method unless it's been defined, and it only calls each unique method once.